src/main/java/no/nav/tag/tiltaksgjennomforing/avtale/Arbeidsgiver.java

```diff
@@ -160,6 +171,16 @@ public boolean avtalenEksisterer(Avtale avtale) {
         return gyldigeTilgangerPåBedriftNr.contains(tiltakstype);
     }
 
+    private boolean harTilgangPåDeltakerIBedrift(BedriftNr bedriftNr, Fnr deltakerFnr) {
+        if (!adressesperreTilgang.contains(bedriftNr)) {
+            Diskresjonskode diskresjonskode = persondataService.hentDiskresjonskode(deltakerFnr);
+            if (diskresjonskode.erKode6Eller7()) {
+                return false;
+            }
+        }
+        return true;
```

Comment:
Det blir kanskje enklere å lese uten nøsting av if'ene?

```suggestion
        if (adressesperreTilgang.contains(bedriftNr)) {
            return true;
        }
        
        Diskresjonskode diskresjonskode = persondataService.hentDiskresjonskode(deltakerFnr);
        return !diskresjonskode.erKode6Eller7();
```

Reply:
litt usikker..